framework/auth/core.py

```diff
@@ -286,13 +286,15 @@ class User(GuidStoredObject, AddonModelMixin):
     # }
 
     # email lists to which the user has chosen a subscription setting
-    mailing_lists = fields.DictionaryField()
+    mailchimp_mailing_lists = fields.DictionaryField()
     # Format: {
     #   'list1': True,
     #   'list2: False,
     #    ...
     # }
 
+    osf_mailing_lists = fields.DictionaryField(default={'Open Science Framework Help': True})
```

Comment:
Please add a comment explaining what this field is, as well as it's format. With so many different emails in the OSF it's easy to get them confused.